src/components/SignIn/SignIn.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable linebreak-style */
+import React, { useRef, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link, Navigate } from 'react-router-dom';
+import { postSignIn } from '../../redux/registration/registration';
+import { isAuthenticated } from '../../redux/current_user/current_user';
+import './signin.css';
+
+const SignIn = () => {
+  const dispatch = useDispatch();
+  const formRef = useRef();
+  const [loading, setLoading] = useState(false);
+  const authenticated = isAuthenticated();
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const formData = new FormData(formRef.current);
+    const data = Object.fromEntries(formData);
+    const user = {
+      email: data.email,
+      password: data.password,
+    };
+    /* eslint-disable no-console */
+    console.log(user);
```

Comment:
- [ ] Kindly remove this old piece of debugging console.log statement, in order to improve the repo's cleanliness. 👍